modules/administration-guide/partials/proc_backing-up-a-che-instance-to-amazon-s3-by-using-prod-cli-with-command-line-options.adoc

```diff
@@ -0,0 +1,41 @@
+[id="backing-up-a-{prod-id-short}-instance-to-amazon-s3-by-using-prod-cli-with-command-line-options_{context}"]
+= Backing up a {prod-short} instance to Amazon S3 by using {prod-cli} with command-line options
+
+To make a backup snapshot of a {prod-short} instance and send the snapshot to the configured Amazon S3, run {prod-cli} with the command-line options.
```

Comment:
```suggestion
To make a backup snapshot of a {prod-short} instance and send the snapshot to the configured Amazon S3 or API compatible storage server, run {prod-cli} with the command-line options.
```

Reply:
@mmorhun, I've added `or S3 API compatible storage` in the assembly file, as you requested. Regarding procedure modules: if you want, we can open a Jira issue to add a separate procedure specifically for using https://min.io/.